chef_master/source/resource_mount.rst

```diff
@@ -41,7 +41,7 @@ The full syntax for all of the properties that are available to the **mount** re
      password                   String
      provider                   Chef::Provider::Mount
      subscribes                 # see description
-     supports                   Array
+     supports                   Hash # defaults to { :remount => false }
```

Comment:
I wanted to add some comments in here around the code based on a conversation I was having someone at Chef. Hash all you want. Array all you want. 😄 I like seeing how code works and talk about it.